base/shell/explorer/syspager.cpp

```diff
@@ -1231,8 +1231,10 @@ void CNotifyToolbar::Initialize(HWND hWndParent, CBalloonQueue * queue)
         TBSTYLE_FLAT | TBSTYLE_TOOLTIPS | TBSTYLE_WRAPABLE | TBSTYLE_TRANSPARENT |
         CCS_TOP | CCS_NORESIZE | CCS_NOPARENTALIGN | CCS_NODIVIDER;
 
-    // HACK & FIXME: CORE-17505
-    SubclassWindow(CToolbar::Create(hWndParent, styles));
+    // HACK & FIXME
+    HWND hwnd = CToolbar::Create(hWndParent, styles);
+    m_hWnd = NULL;
```

Comment:
Is it the `CToolbar::Create` that sets the `m_hWnd` member?

Reply:
`CToolbar::Create` does create the window but it doesn't subclass itself. Maybe its class hierarchy has a bug that the `SubclassWindow` method can't be called from `CToolbar`.